src/utils.js

```diff
@@ -29,6 +29,8 @@ export function getLMSIcon(LMSType) {
       return CornerstoneIcon;
     case DEGREED_TYPE:
       return DegreedIcon;
+    case DEGREED2_TYPE:
+      return DegreedIcon;
     case MOODLE_TYPE:
       return MoodleIcon;
     default:
```

Comment:
do we want SAPIcon to be default? or just null?

Reply:
🤔 I guess null? Both cases are pretty bad and preferably avoidable.

Reply:
I think null is better since it indicates we did not find a matching valid input